.github/workflows/review.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: GPTcode-reviewer
-        uses: PierreGode/GPTcode-reviewer@dev
+        uses: PierreGode/GPTcode-reviewer@PR-LOG
```

Comment:
Using a specific branch like `PR-LOG` may lead to inconsistencies if that branch is not maintained or updated regularly. Consider using a stable release version instead.